tree/dataframe/inc/ROOT/RDataSource.hxx

```diff
@@ -197,6 +197,7 @@ public:
    // clang-format on
    virtual void Initialize() {}
 
+private:
```

Comment:
these should be private, but ROOT coding conventions then demand that these entities be lifted up to the beginning of the class definition (before the first `public:`). The `/// \cond` and `///\ endcond` need to be refactored accordingly

Reply:
Like this? Just learned about the `\cond` / `\endcond` stuff...